packages/admin-ui/ui/src/domain/orders/details/detail-cards/summary.tsx

```diff
@@ -28,9 +34,49 @@ const SummaryCard: React.FC<SummaryCardProps> = ({ order, reservations }) => {
   } = useToggleState()
 
   const { showModal } = useContext(OrderEditContext)
+  const { client } = useMedusa()
   const { isFeatureEnabled } = useFeatureFlag()
   const inventoryEnabled = isFeatureEnabled("inventoryService")
 
+  const [variantInventoryMap, setVariantInventoryMap] = React.useState<
+    Map<string, VariantInventory>
+  >(new Map())
+
+  React.useEffect(() => {
+    if (!inventoryEnabled) {
+      return
+    }
+
+    const fetchInventory = async () => {
+      const inventory = await Promise.all(
+        order.items.map(async (item) => {
+          if (!item.variant_id) {
+            return
+          }
+          return await client.admin.variants.getInventory(item.variant_id)
+        })
+      )
+
+      const a = new Map(
```

Comment:
todo: would be nice to avoid such naming 😂

Reply:
😂 forgot one, thanks